rules/false_positives/trino.yara

```diff
@@ -0,0 +1,13 @@
+rule trino_launcher: override {
+  meta:
+    description                                     = "launcher"
+    SIGNATURE_BASE_SUSP_ELF_LNX_UPX_Compressed_File = "high"
+
+  strings:
+    $go      = "go1.23"
+    $module  = "/Users/martin/go/pkg"
+    $partial = "martin/go"
+
+  condition:
+    filesize <= 1536KB and any of them
```

Comment:
Should this be "all of them"? The go string is fairly broad.

Alternatively, I'm ok with changing the underlying  rule to "high", but maybe not until we switch wolfi to use differential analysis.

Reply:
Each of the launchers only had one of the strings 😞

Reply:
Ok. The post-compression strings are essentially going to be unpredictable. Just mark the underlying rule as "high" & remove the override. 

Critical is supposed to be reserved for cases where there is no question about maliciousness. Here we have a great counter-example. We can leave the critical findings for tampered UPX rules that this program did not hit.

Reply:
Turned the third-party rule off in [`acc4401` (#694)](https://github.com/chainguard-dev/malcontent/pull/694/commits/acc44011543bc24207995781b5a74548f1e79d53).

We can just rely on our UPX rule which is already marked as high.